### PR TITLE
v54.6.1

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
cc @AlimoRabbani

File: components/schemes/Color/src/MDCSemanticColorScheme.h [hunk 1]
```diff
@@ -116,4 +116,14 @@ typedef NS_ENUM(NSInteger, MDCColorSchemeDefaults) {
  */
 - (nonnull instancetype)initWithDefaults:(MDCColorSchemeDefaults)defaults;
 
+/**
```
Comment (romoore) [inline components/schemes/Color/src/MDCSemanticColorScheme.h:119]: Why is this needed for a hotfix?

**Comment by jverkoey** [comment]
This PR is malformed.
